docs/api-reference/carto/fetch-map.md

```diff
@@ -0,0 +1,82 @@
+# fetchMap
+
+`fetchMap` is an API that instantiates layers configured in [CARTO Builder](https://carto.com/builder/) for use with deck.gl. It is available starting with CARTO Maps API version v3.
```

Comment:
This part is more a work for product than engineering. @borja-munoz could you help here? 

This is a little bit confusing, we're linking to the CARTO builder landing page, so people can think it will work in the builder version of CARTO 2, have in mind lots of users don't know about API versions.

We need to rethink what's going to be the product strategy around this feature in our side and doc. 

In the meantime, to avoid blocking this PR, can we add here a quick gif to demo the feature? And remove the link

Reply:
I agree, we should remove the link to the current Builder landing page until we update it in the public website. I think adding a video is a good idea.